core/contracts/financial-templates/implementation/PricelessPositionManager.sol

```diff
@@ -549,20 +563,23 @@ contract PricelessPositionManager is FeePayer, AdministrateeInterface {
      * rawCollateral, which isn't a user-readable value.
      * @param sponsor address whose collateral amount is retrieved.
      */
-    function getCollateral(address sponsor) external view returns (FixedPoint.Unsigned memory) {
+    function getCollateral(address sponsor) external view nonReentrantView() returns (FixedPoint.Unsigned memory) {
         // Note: do a direct access to avoid the validity check.
         return _getFeeAdjustedCollateral(positions[sponsor].rawCollateral);
     }
 
     /**
      * @notice Accessor method for the total collateral stored within the PricelessPositionManager.
      */
-    function totalPositionCollateral() external view returns (FixedPoint.Unsigned memory) {
+    function totalPositionCollateral() external view nonReentrantView() returns (FixedPoint.Unsigned memory) {
         return _getFeeAdjustedCollateral(rawTotalPositionCollateral);
     }
 
     /**
      * @dev This overrides pfc() so the PricelessPositionManager can report its profit from corruption.
+     * Note: We do not apply the nonReentrantView() modifier here because it is applied to `payRegularFees()` which will call `pfc()`.
+     * We could have replaced added an internal `_pfc()` method to be called by `payRegularFees()` in order to make this `nonReentrantView()`,
+     * but decided not to in order to save bytecode.
```

Comment:
Now that we have the bytecode issue solved, should we do this just for consistency? Not super opinionated either way, but just wanted to mention it.

Reply:
Yeah thanks for the reminder. I added the modifier back and it ended up adding 25 bytes to the deployedBytecode FYI.